tests/src/test/java/org/jboss/forge/test/roaster/model/common/JavaClassTestBase.java

```diff
@@ -123,9 +124,10 @@ public void testSetPackageDefault() throws Exception
    }
 
    @Test(expected = IllegalArgumentException.class)
+   @Ignore
```

Comment:
Why is this test ignored?

Reply:
Forgot to remove the @Ignore annotation. Next commit should fix this